common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -112,8 +112,8 @@ public static String getVersion() {
         return version;
     }
 
-    public static boolean isPreAlpha() {
-        return version.contains("pre-alpha");
+    public static boolean isBeta() {
+        return version.contains("beta");
```

Comment:
I am assuming we will be calling the builds beta and not pre-alpha but can revert if not